SpatialGDK/Documentation/content/glossary.md

```diff
@@ -1,33 +1,119 @@
 # Glossary
+This glossary covers:
+* Terms used in this documentation
+* Terms relevant to the GDK for Unreal
+* SpatialOS terms used in the GDK for Unreal documentation
 
-## SpatialOS terms
-See the [SpatialOS glossary](https://docs.improbable.io/reference/latest/shared/glossary) for terms specific to SpatialOS.
+**SpatialOS documentation**<br/>
+Many SpatialOS term definitions link to further information in the [SpatialOS documentation](https://docs.improbable.io/reference/latest/index). This documentation covers the SpatialOS SDK - a software development kit for SpatialOS which underpins the GDK for Unreal, as well some GDK-relevant tools; the [Console](#console), [schema](#schema), [snapshots](#snapshots), and [configuration files](#configuration-files) in particular.
```

Comment:
I agree that we should try to avoid "SpatialOS SDK" here, as it's not a term we really use anywhere else (yet). Plus, we then go on to talk about the "SpatialOS SDKs" (plural) - i.e. the Worker SDK and Platform SDK - which makes it more confusing.

Maybe leave out "SpatialOS SDK" and just mention the Worker and Platform SDKs?

```suggestion
Many SpatialOS term definitions link to further information in the [SpatialOS documentation](https://docs.improbable.io/reference/latest/index). This documentation covers the SpatialOS Worker SDK and Platform SDK, as well some GDK-relevant tools; the [Console](#console), [schema](#schema), [snapshots](#snapshots), and [configuration files](#configuration-files) in particular.
```